### DISCUSSION TITLE
Support for a Optuna framework?

### DISCUSSION THREAD

**Discussion by kurucan** [microsoft/nni#3802] [Ideas]
Any plans to support Optuna hyperparameter optimization framework?

**Comment by scarlett2018** [comment]
investigation needed. @kurucan - are you using Optuna HO framework today? how are you seeing it been supported by NNI?

**Comment by QuanluZhang** [comment]
@kurucan , which part of Optuna do you want to be supported by NNI, the tuning algorithms in Optuna?

**Reply by scarlett2018** [reply]
ping @kurucan for more information about this new feature request.
